pkgs/servers/mail/mailman/web.nix

```diff
@@ -1,16 +1,16 @@
 { buildPythonPackage, lib, fetchgit, isPy3k
-, git, makeWrapper, sassc, hyperkitty, postorius, whoosh
+, git, makeWrapper, sassc, hyperkitty, postorius, whoosh, setuptools-scm
 }:
 
 buildPythonPackage rec {
   pname = "mailman-web-unstable";
-  version = "2019-09-29";
+  version = "2021-04-10";
```

Comment:
```suggestion
  version = "unstable-2021-04-10";
```

Reply:
Is this necessary? The package name has unstable. I feel if we're doing this we should remove "unstable" from the package name.

Reply:
> Is this necessary? The package name has unstable.

If we only have one mailman-web package in nixpkgs then the unstable should be moved to the version.